assets/styles.css

```diff
@@ -0,0 +1,399 @@
+/* CSS Reset */
+
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
+
+a {
+  text-decoration: none;
+  color: black;
+  font-size: 14px;
+}
+
+h1 {
+  font-size: 40px;
+}
+
+small {
+  font-size: 10px;
+}
+
+p {
+  margin-top: 20px;
+  font-size: 20px;
+  color: #333;
+}
+
+p small {
+  font-size: 12px;
+}
+
+.header {
+  position: fixed;
+  top: 0;
+  z-index: 1;
+  background-color: #fff;
+  box-shadow: 0 1px 1px rgba(0, 0, 0, 0.1);
+  padding: 0 20px;
+  display: flex;
+  align-items: center;
+}
+
+.header,
+.section-one img,
+.inflation img,
+.grid-item img,
+.sticky-footer {
+  width: 100%;
+}
+
+.first-section,
+.second-section,
+.third-section {
+  width: 33.33%;
+  float: left;
+}
+
+.more-comments button {
+  width: 250px;
+  padding: 10px;
+  border: none;
+  border-radius: 3px;
+  background-color: #567b95;
+  color: #fff;
+}
+
+.sticky-footer button {
+  position: absolute;
+  right: 10vw;
+  bottom: 20px;
+  background: transparent;
+  border: none;
+  text-transform: uppercase;
+  color: #333;
+}
+
+.third-section button {
+  background-color: #567b95;
+  color: #fff;
+  padding: 7px 10px;
+  border-color: #567b95;
+  border-radius: 3px;
+  text-transform: uppercase;
+  font-weight: bold;
+  font-size: 11px;
+}
+
+.first-section .fa:hover,
+.third-section button:hover,
+.sticky-footer button:hover {
+  cursor: pointer;
+}
+
+.second-section {
+  text-align: center;
+}
+
+.second-section img {
+  width: 220px;
+}
+
+.third-section {
+  text-align: right;
+}
+
+main {
+  position: relative;
+  top: 60px;
+}
+
+.section-one {
+  width: 70vw;
+  margin: 30px auto;
+}
+
+.section-one h1 {
+  width: 50%;
+  margin: 20px auto;
+}
+
+.caption {
+  color: #666;
+  font-size: 16px;
+}
+
+article header a {
+  font-size: 16px;
+}
+
+.article-description {
+  width: 50%;
+  margin: 30px auto;
+  position: relative;
+}
+
+.date-socials {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin-top: 10px;
+}
+
+.fa-facebook,
+.fa-twitter,
+.fa-envelope,
+.fa-share {
+  background-color: #f5f5f5;
+  padding: 5px 10px;
+  border-radius: 15px;
+}
+
+.date {
+  font-size: 14px;
+}
+
+.section-title {
+  font-size: 20px;
+  margin-top: 20px;
+}
+
+.subscribe-section {
+  margin-top: 20px;
+  text-align: center;
+}
+
+.subscribe-section hr {
+  width: 90px;
+  margin: 20px auto;
+}
+
+.subscribe-section a {
+  font-size: 1em;
+  position: relative;
+  top: 3px;
+  color: #326891;
+}
+
+.aside-section {
+  position: absolute;
+  width: 220px;
+  top: 1700px;
+  right: 40px;
+}
+
+.inflation img {
+  margin-top: 10px;
+}
+
+.spiral-image img {
+  width: 500px;
+}
+
+.youtube-section a {
+  font-size: 20px;
+  color: #326891;
+}
+
+.footer-header img {
+  position: relative;
+  float: left;
+  width: 200px;
+}
+
+.footer-header a {
+  position: relative;
+  float: right;
+  text-align: right;
+  color: #567b95;
+  font-size: 12px;
+  width: 200px !important;
+}
```

Comment:
## No `!important` overusing[1]

The !important rule was conscripted in the mid-to-late 90’s to help web designers and developers override the rules of their style sheet. 

- [ ] Remove all `!important` rules in your entire CSS file

<hr>

_[1] [Good and bad  CSS practices](https://speckyboy.com/good-bad-css-practices/)_